<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="6" passed="5">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="51158" started-at="2017-11-24T10:42:38Z" finished-at="2017-11-24T10:43:29Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="51158" started-at="2017-11-24T10:42:38Z" finished-at="2017-11-24T10:43:29Z">
      <class name="com.ci.demo.MobileApp_Test">
        <test-method status="PASS" signature="mobileTestCase()[pri:0, instance:com.ci.demo.MobileApp_Test@addb54]" name="mobileTestCase" duration-ms="51142" started-at="2017-11-24T10:42:38Z" finished-at="2017-11-24T10:43:29Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- mobileTestCase -->
      </class> <!-- com.ci.demo.MobileApp_Test -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
  <suite name="Suite (1)" duration-ms="3663" started-at="2017-11-24T10:43:37Z" finished-at="2017-11-24T10:43:41Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="3663" started-at="2017-11-24T10:43:37Z" finished-at="2017-11-24T10:43:41Z">
      <class name="com.ci.demo.WebservicesTest">
        <test-method status="FAIL" signature="webserviceTestcase()[pri:0, instance:com.ci.demo.WebservicesTest@17e1986]" name="webserviceTestcase" duration-ms="3663" started-at="2017-11-24T10:43:37Z" finished-at="2017-11-24T10:43:41Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertTrue(Assert.java:41)
	at org.testng.Assert.assertTrue(Assert.java:51)
	at com.ci.demo.WebservicesTest.webserviceTestcase(WebservicesTest.java:44)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:281)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:121)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- webserviceTestcase -->
      </class> <!-- com.ci.demo.WebservicesTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite (1) -->
  <suite name="Suite (0)" duration-ms="8254" started-at="2017-11-24T10:43:29Z" finished-at="2017-11-24T10:43:37Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="8254" started-at="2017-11-24T10:43:29Z" finished-at="2017-11-24T10:43:37Z">
      <class name="com.ci.demo.WebTesting">
        <test-method status="PASS" signature="launchBrowser()[pri:0, instance:com.ci.demo.WebTesting@14bef66]" name="launchBrowser" duration-ms="8251" started-at="2017-11-24T10:43:29Z" finished-at="2017-11-24T10:43:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchBrowser -->
      </class> <!-- com.ci.demo.WebTesting -->
    </test> <!-- Test -->
  </suite> <!-- Suite (0) -->
  <suite name="Suite (2)" duration-ms="16" started-at="2017-11-24T10:43:41Z" finished-at="2017-11-24T10:43:41Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="16" started-at="2017-11-24T10:43:41Z" finished-at="2017-11-24T10:43:41Z">
      <class name="com.ci.demo.math.Multiplication_Table_Test">
        <test-method status="PASS" signature="testCase2()[pri:0, instance:com.ci.demo.math.Multiplication_Table_Test@1fb3ab6]" name="testCase2" duration-ms="16" started-at="2017-11-24T10:43:41Z" finished-at="2017-11-24T10:43:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCase2 -->
      </class> <!-- com.ci.demo.math.Multiplication_Table_Test -->
      <class name="com.ci.demo.math.PalindromeLessThanThousand_Test">
        <test-method status="PASS" signature="testCase2()[pri:0, instance:com.ci.demo.math.PalindromeLessThanThousand_Test@76ae34]" name="testCase2" duration-ms="0" started-at="2017-11-24T10:43:41Z" finished-at="2017-11-24T10:43:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCase2 -->
      </class> <!-- com.ci.demo.math.PalindromeLessThanThousand_Test -->
      <class name="com.ci.demo.math.PrintPrimeNumbers_Test">
        <test-method status="PASS" signature="testCase1()[pri:0, instance:com.ci.demo.math.PrintPrimeNumbers_Test@146e1f4]" name="testCase1" duration-ms="0" started-at="2017-11-24T10:43:41Z" finished-at="2017-11-24T10:43:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCase1 -->
      </class> <!-- com.ci.demo.math.PrintPrimeNumbers_Test -->
    </test> <!-- Test -->
  </suite> <!-- Suite (2) -->
</testng-results>
